checkout.py

```diff
@@ -18,13 +21,13 @@
         else:
             items.append(item_id_input)
     else:
-        print ("That is an invalid product identifier. You are fired!")
+        print("That is an invalid product identifier. You are fired!")
 
-print (items)
+print(items)
 
 def create_custom_receipt():
     dashes = "--------------------------------"
-    return dashes + "\n" + GROCERY_STORE_NAME + "\n" + dashes
+    return dashes + "\n" + " "*9 + GROCERY_STORE_NAME + "\n\n" + DATE_TIME_FORMAT + "\n" + dashes
```

Comment:
I am thinking that this line of code is going to get out of hand pretty soon - 
it might make the code easier to understand if we put the strings that are on different lines on the receipt onto different lines here. Like this:

```
return dashes + "\n" 
            + " "*9 + GROCERY_STORE_NAME + "\n\n" 
            + DATE_TIME_FORMAT + "\n" 
            + dashes

```

Reply:
Great point thank you so much